src/net/Lib/Clients.cs

```diff
@@ -23,6 +24,7 @@ public class Clients : IAsyncDisposable
         public BlobContainerClient ContainerClient;
         public IMessagingProvider MessagingProvider;
         public TextAnalyticsClient TextAnalyticsClient;
+        public IImageProvider ImageProvider { get; private set; }

```

Comment:
SHould we do get/set for the rest of the properties?

Reply:
no - that was on me